lib/logstash/codecs/telemetry.rb

```diff
@@ -42,18 +43,17 @@ module TSCodecTLVType
 #
 # IOS XR 6.1.0 wire format (version 2)
 #
-# JSON Telemetry encoding
+# JSON and GPB Telemetry encoding
 #
 # ----------------------------------
 #
 #       +-+-+-+-...-++-+-+-+-...-++-+-+-+-...+
 #       |T|F|L|V... ||T|F|L|V... ||T|F|L|V...|
 #       +-+-+-+-...-++-+-+-+-...-++-+-+-+-...+
 #
-# T   - 32 bits, type (JSON == 2), currently the only supported value (GPB
-#       variant over TCP not supported by codec yet) TSCodecTLVTypeV2
-# F   - 32 bits, 0x1 indicates ZLIB compression
-# L   - Length of block excluding header
+# T   - 32 bits, type (GPB compact == 3, GPB kv == 4)
+# F   - 32 bits, 0x0 No flags set – default behavior - nocompression
```

Comment:
Why the change here? Compression can still be set, but you will want to indicate whether and  under which circumstances if any we still support compression,